lib/LedgerSMB/Sysconfig.pm

```diff
@@ -504,6 +520,27 @@ sub check_permissions {
     }
 }
 
+# if you have latex installed set to 1
+###TODO-LOCALIZE-DOLLAR-AT
+our $latex = 0;
+
+
+sub override_defaults {
+    # Check Latex
+    $latex = eval {require Template::Plugin::Latex; 1;};
+
+    # Check availability and loadability
+    $LedgerSMB::Sysconfig::template_latex = eval {require LedgerSMB::Template::LaTeX; 1}
```

Comment:
I think we should, but this is broader than the scope of this PR, as it implies moving cpanfile checks to runtime and should be done for all modules.